executor/src/witgen/machines/machine_extractor.rs

```diff
@@ -309,6 +336,9 @@ fn all_row_connected_witnesses<T>(
                         witnesses.extend(in_rhs);
                     }
                 }
+                Identity::Permutation(..) | Identity::Connect(..) => {
+                    todo!("same as plookup, avoid repetition?")
+                }
```

Comment:
The comment of this function does not mention connect, is this something we use?

Reply:
Connect is very sporadically implemented.